frontend/src/constants/screens.ts

```diff
@@ -1,9 +1,4 @@
 /**
- * This file cannot be converted to TypeScript because it's referenced in the
- * Tailwind config. `RealBreakpoints` excludes 'xs' because it's not defined in
- * Tailwind (being mobile-first) but is used in JavaScript to refer to mobile
- * screens that are narrower than 640px ('sm').
- *
  * @typedef {'2xl' | 'xl' | 'lg' | 'md' | 'sm' | 'xs'} Breakpoint
```

Comment:
These types should be rewritten using TS:
~~`* @typedef {'2xl' | 'xl' | 'lg' | 'md' | 'sm' | 'xs'} Breakpoint`~~
`export type Breakpoint = '2xl' | 'xl' | 'lg' | 'md' | 'sm' | 'xs'`